app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+from abc import ABC
```

Comment:
The same as in dwarf module. Additionally I wand to remind you that all in imports in every section should be sort in alphabetical order